README.md

```diff
@@ -1 +1,32 @@
-# riscv
+﻿RV32G Processor model
+===================================================================
+
+This is the functional model of RISC-V instruction set architecture. It contains all the instructions necessary for running mibench programs. It has following types of instruction encoding:
+1) Integer Base instructions
+2) Single-Precision floating point instructions
+3) Double-Precision floating point instructions
+
+Installation
+------------------------------------------------------------------
+For installing the processor model follow the below steps:
+Step 1:. Download the RISC-V processor model in the processors folder(made during the installation of ArchC)
+Step 2:. Perform the following commands on terminal:
+`````````
+	acsim riscv.ac -abi		(Create the simulator)
+	make					(Compile it)
+`````````
+
+How to create executables
+-------------------------------------------------------------------
+Follow the below steps:
+Step 1:. Go to the tests->libac_sysc folder and run make command via terminal.
+Step 2:. Copy the folder of the program you wish to test from test->acstone-prgrams or test->automotive-IMA to the test folder. Enter that folder and run make via terminal.
```

Comment:
Which folder is this? The Makefile runs the riscv compiler with a special input file encoded in a relative path  "../rv_hal/crt.S" that I don't have in my system because it is not present in the riscv64 compiler downloaded from their website. Where do I get this file, which seems to be necessary to compile the programs?